tests/storage/snapshot.go

```diff
@@ -1293,49 +1293,59 @@ var _ = SIGDescribe("VirtualMachineSnapshot Tests", func() {
 
 				Expect(snapshot.Status.CreationTime).To(BeNil())
 			})
+		})
 
-			It("vmsnapshot should update error if vmsnapshotcontent is unready to use and error", func() {
-				vm = tests.RunVMAndExpectLaunchWithRunStrategy(virtClient, vm, v1.RunStrategyAlways)
-				// Delete DV and wait pvc get deletionTimestamp
-				// when pvc is deleting snapshot is not possible
-				volumeName := vm.Spec.DataVolumeTemplates[0].Name
-				By("Deleting Data volume")
-				err = virtClient.CdiClient().CdiV1beta1().DataVolumes(vm.Namespace).Delete(context.Background(), volumeName, metav1.DeleteOptions{})
-				Expect(err).ToNot(HaveOccurred())
-				Eventually(func() *metav1.Time {
-					pvc, err := virtClient.CoreV1().PersistentVolumeClaims(vm.Namespace).Get(context.Background(), volumeName, metav1.GetOptions{})
-					Expect(err).ToNot(HaveOccurred())
-					return pvc.DeletionTimestamp
-				}, 30*time.Second, time.Second).ShouldNot(BeNil())
+		It("vmsnapshot should update error if vmsnapshotcontent is unready to use and error", func() {
+			vm = renderVMWithRegistryImportDataVolume(cd.ContainerDiskAlpine, snapshotStorageClass)
+			runStrategyAlways := v1.RunStrategyAlways
```

Comment:
I a future PR you can replace this with `libvmi.WithRunStrategy(v1.RunStrategyAlways)(vm)` or add `libvmi.Option...` to `renderVMWithRegistryImportDataVolume`